chromadb/test/property/test_collections.py

```diff
@@ -89,6 +89,21 @@ def list_collections(self) -> None:
         for c in colls:
             assert c.name in self.model
 
+    # @rule for list_collections with limit and offset
+    @rule(
+        limit=st.integers(min_value=1, max_value=5),
+        offset=st.integers(min_value=0, max_value=5),
+    )
+    def list_collections_with_limit_offset(self, limit: int, offset: int) -> None:
+        colls = self.api.list_collections(limit=limit, offset=offset)
```

Comment:
We should validate this list has the contents we expect, this would not catch that.

Reply:
very good point - will improve

Reply:
this is done